sourcegraph/browser-extensions
# refactor: update to use new cxp and @s/e-c-c deps

Updates the cxp and @sourcegraph/extensions-client-common dependencies and the code in this repository using them to account for the changes in the following PRs:

https://github.com/sourcegraph/cxp-js/pull/15
https://github.com/sourcegraph/cxp-js/pull/18
https://github.com/sourcegraph/extensions-client-common/pull/6
https://github.com/sourcegraph/extensions-client-common/pull/4

Testing plan:

1. Run with the CXP feature flag on, ensure that the Codecov extension works
2. Run with the CXP feature flag off, ensure that hovers work

I tested it on the file https://github.com/theupdateframework/notary/blob/master/server/storage/tuf_store.go.

I have tested on:
